pkg_api/server/config.py

```diff
@@ -0,0 +1,44 @@
+"""Define server configuration."""
+
+
+from pkg_api.nl_to_pkg.annotators.three_step_annotator import (
+    _DEFAULT_CONFIG_PATH as DEFAULT_3_STEP_CONFIG_PATH,
+)
+from pkg_api.nl_to_pkg.annotators.three_step_annotator import (
+    _DEFAULT_PROMPT_PATHS,
+)
+from pkg_api.nl_to_pkg.entity_linking.spotlight_entity_linker import (
+    _DEFAULT_CONFIG_PATH as DEFAULT_SPOTLIGHT_CONFIG_PATH,
+)
+from pkg_api.pkg import DEFAULT_VISUALIZATION_PATH
+
+
+class BaseConfig:
+    """Base configuration for the server."""
+
+    TESTING = False
+
+    # Three step annotator configuration
+    TS_ANNOTATOR_CONFIG_PATH = DEFAULT_3_STEP_CONFIG_PATH
+    TS_ANNOTATOR_RPOMPT_PATHS = _DEFAULT_PROMPT_PATHS
+
+    # Entity linker configuration
+    SPOTLIGHT_CONFIG_PATH = DEFAULT_SPOTLIGHT_CONFIG_PATH
+
+
+class DevelopmentConfig(BaseConfig):
+    """Development configuration for the server."""
+
+    DEBUG = True
+    SQLALCHEMY_DATABASE_URI = "sqlite:///db.sqlite"
```

Comment:
Note that this is the configuration file used for the server. According to [Flask documentation](https://flask.palletsprojects.com/en/2.3.x/config/), the configuration can be defined as a class. Here we consider three configurations, the base one contain shared parameter, while the development and testing one change the path for storage to avoid mixing the files and facilitate cleaning.